[Bug] [Tools] DB configurations not work for tools

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

* If we set database to `mysql` in `dolphinscheduler_env.sh`, when running `upgrade-schema.sh` in `tools`, will get errors like `Caused by: java.lang.RuntimeException: Driver com.mysql.cj.jdbc.Driver claims to not accept jdbcUrl, jdbc:postgresql://127.0.0.1:5432/dolphinscheduler`
* `datasource.url` is somehow deleted in PR: #10128 , causing the datasource url will not be overwritten if `mysql` profile activated. I'm confused why that line was deleted in that PR.
* https://github.com/apache/dolphinscheduler/blob/9782fe4e7cd0c9313c748bb45b5b1f2d1ddd3367/dolphinscheduler-tools/src/main/resources/application.yaml#L39-L46

![image](https://user-images.githubusercontent.com/34905992/172893278-e54e4046-0d6b-4328-9251-03117d82582f.png)


### What you expected to happen

* Already described above.

### How to reproduce

* Already described above.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-tools/src/main/resources/application.yaml
+++ b/dolphinscheduler-tools/src/main/resources/application.yaml
@@ -43,6 +43,7 @@ spring:
       on-profile: mysql
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
 
 ---
 spring:

